.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Create a report to help us improve Juju
+labels:
+  - kind/bug
+  - needs-triage
+body:
+  - type: textarea
+    id: Description
+    attributes:
+      label: Description
+      description: "Please describe the bug for us: "
+      placeholder: ex. "`juju status` returns the error `not found`"
+    validations:
+      required: true
+
+  - type: input
+    id: Version
+    attributes:
+      label: Juju version
+      description: "What version of Juju are you using?"
+      placeholder: ex. 3.6.1
+    validations:
+      required: true
+
+  - type: input
+    id: Cloud
+    attributes:
+      label: Cloud
+      description: What cloud are you using?
+      placeholder: ex. AWS
+    validations:
+      required: false
+
+  - type: textarea
+    id: Expected-Behaviour
+    attributes:
+      label: Expected behaviour
+      description: "Describe of what you expected to happen: "
+      placeholder: ex. "`juju status` should tell me what was not found"
+    validations:
+      required: true
+
+  - type: textarea
+    id: Reproduction-Steps
+    attributes:
+      label: "Reproduce / Test"
+      description: "Steps to reproduce the behaviour: "
+      placeholder: ex. "juju bootstrap aws && juju status"
+    validations:
+      required: false
+
+  - type: textarea
+    id: Info-Notes
+    attributes:
+      label: "Notes & References"
+      description: "Please add relevant notes, other related issues/PRs, anything to help diagnose and solve the issue."
+    validations:
+      required: false
```

Comment:
General: should we be more specific in what we seek as info? e.g. instruct them to attach some log files? messages? something that is easily available.